@here/harp-map-theme/resources/berlin_tilezen_base.json

```diff
@@ -1182,6 +1189,56 @@
                 },
                 "final": true
             },
+            {
+                "description": "POIs in tilezen format use maki icons",
+                "when": "$layer == 'pois' && has(kind)",
+                "technique": "labeled-icon",
+                "attr": {
+                    "poiTable": "tzPoiTableMaki",
+                    "poiNameField": "kind",
+                    "imageTexturePostfix": "-15",
+                    "iconScale": 1,
+                    "size": 16,
+                    "yOffset": 8,
+                    "vAlignment": "Above",
+                    "hAlignment": "Center",
+                    "textIsOptional": true,
+                    "iconIsOptional": false,
+                    "renderTextDuringMovements": false,
+                    "textMayOverlap": false,
+                    "textReserveSpace": true,
+                    "iconMayOverlap": true,
+                    "iconReserveSpace": false,
+                    "color": "#090A0B",
+                    "backgroundColor": "#FFFFFF",
+                    "backgroundOpacity": 0.5
+                },
+                "final": true
+            },
+            {
+                "description": "POIs incompatible to tilezen format - draw debug",
```

Comment:
remove all "draw debug" and "no kind" styles.

Reply:
This was actually not my idea, but something what is propagated in every theme, usually in worse form (without correct description, etc..).... So are you sure we don't want it anymore?